packages/in_app_purchase/lib/src/store_kit_wrappers/sk_product_wrapper.dart

```diff
@@ -146,6 +146,8 @@ class SKProductDiscountWrapper {
   /// The [subscriptionPeriod] of the discount is independent of the product's [subscriptionPeriod],
   /// and their units and duration do not have to be matched.
   final SKProductSubscriptionPeriodWrapper subscriptionPeriod;
+
+  static _priceFromJson(double price) => num.parse(price.toStringAsFixed(2));
```

Comment:
Are we sure we always want to round the price to two places? Just want to make sure that we're not losing precision here.

Reply:
Ok, I went to check the app store connect and seems like some currencies do have 3-place decimals. Will need to think about how to handle this in a different way.